Panic occurs with `runtime error: index out of range [0] with length 0` in `parseParams` function

Version: 373ed8231a875c5f94aa58ac20300274a818bbcd

I encountered a panic with the error message `runtime error: index out of range [0] with length 0` in the `parseParams` function.

Here's the detailed stack trace:

```
panic: runtime error: index out of range [0] with length 0

goroutine 1 [running]:
github.com/budougumi0617/nrseg.parseParams(0xc000497950?)
    /Users/mpyw/.asdf/installs/golang/1.19/packages/pkg/mod/github.com/budougumi0617/nrseg@v0.0.5/process.go:234 +0x258
...
```

From the investigation, it appears that the error is due to an assumption that the function parameters in the AST always have names, which is not the case. In Go, function parameters can be unnamed. The panic occurs when we try to access `f.Names[0].Name` where `f.Names` has a length of 0, i.e., the parameter is unnamed.

Here's an example of such a case:

```go
func foo(int, string) {}
```

In this function foo, two parameters are unnamed, and when converted into the AST, their corresponding `*ast.Field`'s `Names` field length is 0.

To prevent this, it's necessary to add a check for the existence of the parameter name before trying to access it.

--- a/process.go
+++ b/process.go
@@ -237,14 +237,18 @@ func parseParams(is []*ast.ImportSpec, t *ast.FuncType) (string, string) {
 	for _, f := range t.Params.List {
 		if se, ok := f.Type.(*ast.SelectorExpr); ok {
 			if idt, ok := se.X.(*ast.Ident); ok && idt.Name == cname && se.Sel.Name == "Context" {
-				return f.Names[0].Name, TypeContext
+				if len(f.Names) > 0 && len(f.Names[0].Name) > 0 {
+					return f.Names[0].Name, TypeContext
+				}
 			}
 		}
 		if se, ok := f.Type.(*ast.StarExpr); ok {
 			if se, ok := se.X.(*ast.SelectorExpr); ok {
 				if idt, ok := se.X.(*ast.Ident); ok && idt.Name == hname && se.Sel.Name == "Request" {
-					n = f.Names[0].Name
-					typ = TypeHttpRequest
+					if len(f.Names) > 0 && len(f.Names[0].Name) > 0 {
+						n = f.Names[0].Name
+						typ = TypeHttpRequest
+					}
 				}
 			}
 		}

